src/tests/suite/daos_test.h

```diff
@@ -439,7 +439,13 @@ test_rmdir(const char *path, bool force)
 		if (!strcmp(ent->d_name, ".") || !strcmp(ent->d_name, ".."))
 			continue;   /* skips the dots */
 
-		sprintf(fullpath, "%s/%s", path, ent->d_name);
+		rc = snprintf(fullpath, PATH_MAX, "%s/%s", path, ent->d_name);
```

Comment:
not a big deal, but need to check rc < 0, probably better to be sth like:
if (rc < 0)
    rc = d_errno2der(rc);
if (rc >= PATH_MAX || rc == 0)
     rc = -DER_INVAL;
.....

Reply:
I think rc==0 is a valid return for case that the input src strings are NULL.